servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -246,36 +382,27 @@ def _get_json_batch(self, url, offset, batch_size, timeout, auth):
         return self._get_json(limited_url, timeout, auth)
 
     def _get_json(self, url, timeout, auth=None, verify=True):
-        tags = ["url:%s" % url]
-        msg = None
-        status = None
-        resp = None
-        try:
-            resp = requests.get(url, timeout=timeout, auth=auth, verify=verify)
-            if resp.status_code != 200:
-                status = AgentCheck.CRITICAL
-                msg = "Got %s when hitting %s" % (resp.status_code, url)
-        except requests.exceptions.Timeout:
-            # If there's a timeout
-            msg = "%s seconds timeout when hitting %s" % (timeout, url)
-            status = AgentCheck.CRITICAL
-        except Exception as e:
-            msg = str(e)
-            status = AgentCheck.CRITICAL
-        finally:
-            if status is AgentCheck.CRITICAL:
-                self.service_check(self.SERVICE_CHECK_NAME, status, tags=tags,
-                                   message=msg)
-                raise CheckException("Cannot connect to ServiceNow CMDB, please check your configuration.")
-
-        if resp.encoding is None:
-            resp.encoding = 'UTF8'
+        execution_time_exceeded_error_message = 'Transaction cancelled: maximum execution time exceeded'
+
+        response = requests.get(url, timeout=timeout, auth=auth, verify=verify)
+        if response.status_code != 200:
+            raise CheckException("Got %s when hitting %s" % (response.status_code, url))
+
         try:
-            resp = yaml.safe_load(resp.text.encode("utf-8"))
-        except Exception as e:
-            self.log.exception(str(e))
-            raise CheckException("Exception occurred while parsing response and the error is : {}".format(str(e)))
+            response_json = json.loads(response.text.encode('utf-8'))
+        except json_parse_exception as e:
```

Comment:
I think `response.text.encode('utf-8')` can throw an error that is not an json_parse_exception, we should also catch that

Reply:
Added new exception